alibi_detect/od/sr.py

```diff
@@ -5,15 +5,31 @@
 
 logger = logging.getLogger(__name__)
 
+# numerical stability for division
 EPSILON = 1e-8
 
+# padding values
+PADDING_CONSTANT = 'constant'
+PADDING_REPLICATE = 'replicate'
+PADDING_REFLECT = 'reflect'
+PADDINGS = [PADDING_CONSTANT, PADDING_REFLECT, PADDING_REPLICATE]
+
+# padding sides
+SIDE_BILATERAL = 'bilateral'
+SIDE_LEFT = 'left'
+SIDE_RIGHT = 'right'
+SIDES = [SIDE_BILATERAL, SIDE_RIGHT, SIDE_LEFT]
```

Comment:
Might be slightly nicer to make these enums, inheriting from both `str` and `enum.Enum`, e.g. `class Padding(str, Enum)` so then can refer to e.g. `Padding.CONSTANT` and you could validate user passed strings against allowed values using one of the methods here: https://stackoverflow.com/questions/29503339/how-to-get-all-values-from-python-enum-class/29503414